applications/nrf5340_audio/src/modules/led.c

```diff
@@ -89,6 +94,46 @@ static int config_led_monochrome(uint8_t led_unit, uint8_t led)
 	return gpio_pin_configure_dt(led_units[led_unit].type.mono, GPIO_OUTPUT_INACTIVE);
 }
 
+static void led_units_mapping(enum led_unit_assignment led_unit, uint8_t led)
+{
+	if (led == LED_AUDIO_DEVICE_TYPE_INDEX) {
+		led_units_map[LED_AUDIO_DEVICE_TYPE] = &led_units[led_unit];
+		led_units_num++;
+	} else if (led == LED_AUDIO_NET_STATUS_INDEX) {
+		led_units_map[LED_AUDIO_NET_STATUS] = &led_units[led_unit];
+		led_units_num++;
+	} else if (led == LED_AUDIO_CONN_STATUS_INDEX) {
+		led_units_map[LED_AUDIO_CONN_STATUS] = &led_units[led_unit];
+		led_units_num++;
+	} else if (led == LED_AUDIO_SYNC_STATUS_INDEX) {
+		led_units_map[LED_AUDIO_SYNC_STATUS] = &led_units[led_unit];
+		led_units_num++;
+	} else if (led == LED_AUDIO_APP_STATUS_INDEX) {
+		led_units_map[LED_AUDIO_APP_STATUS] = &led_units[led_unit];
+		led_units_num++;
+	} else {
+		LOG_DBG("LED %d not assigned", led);
+	}
+}
+
+static int find_unit_number(const char *label, uint8_t *unit)
+{
+	uint8_t len = strlen(label);
+
+	for (uint8_t i = 0; i < len; i++) {
+		char c = *label;
+
+		if (isdigit(c)) {
```

Comment:
What happens here if the number consists of two digits?

Reply:
This finds the first digit and then the strtoul() will build the number from however many consecutive digits there are.